README.md

```diff
@@ -175,6 +175,7 @@ fn main() {
 - [Lemoa](https://github.com/lemmy-gtk/lemoa) - Desktop client for Lemmy
 - [Score Tracker](https://github.com/weclaw1/score-tracker) - App for tracking player scores in card and board games
 - [Spidey](https://github.com/kdwk/Spidey) - A seamless and distraction-free work and play environment on the web
+- [Toolbox Tuner](https://github.com/13hannes11/toolbox-tuner) - An application to manage containertoolboxes
```

Comment:
I don't think "containertoolboxes" is a proper word in English. In some languages like German that would make sense, but English doesn't allow adding words together like that. Also, the project is called "Toolbx" despite the CLI being named "toolbox" and it should be more clear if we use the actual project name.

```suggestion
- [Toolbox Tuner](https://github.com/13hannes11/toolbox-tuner) - An application to manage Toolbx containers
```